test/test_operations.py

```diff
@@ -2021,6 +2021,32 @@ def add_one_pallas(output, inputs, payload):
     compiled_add_one_pallas(output, [x], payload)
     self.assertTrue(torch.allclose(output.cpu(), expected_output.cpu()))
 
+  @unittest.skipIf(xr.device_type() != 'TPU', "This test only works on TPU.")
+  def test_tpu_custom_call_pallas_extract_add_payload(self):
```

Comment:
Could the custom kernel tests should be in a separate test file? IMO `test_operations` shouldn't depend on anything other than our basic requirements

* `torchvision` is imported but we don't actually use it

Reply:
yea I agree that pallas related stuff is better to be putted in a new test file.

Reply:
I'm just too lazzzy...

Reply:
lol the problem of the pallas is it introcued the JAX dependency. We might have to disable the test on TPUCI sometimes, it is better not to disable the whole `test_operations.py`

Reply:
Moved to a new file.